src/resolvers/frontpageResolvers.ts

```diff
@@ -79,6 +79,10 @@ export const resolvers = {
       const contexts: TaxonomyContext[] = await queryContexts(id, context);
       return contexts?.find((ctx) => ctx.rootId === "urn:subject:20")?.path || "";
     },
+    async url(id: string, _: any, context: ContextWithLoaders): Promise<string> {
+      const contexts: TaxonomyContext[] = await queryContexts(id, context);
+      return contexts?.find((ctx) => ctx.rootId === "urn:subject:20")?.url || "";
```

Comment:
Hva er greia her? Skal vi ikke rive oss vekk fra `urn:subject:20` og co?

Reply:
Og skal `url` kun funke for film?

Reply:
Tja. Vi snakka jo om det, men enn så lenge er subject:20 === ndla-film.
Og url hentes for node allerede. Var kun filmer som mangla det. Trengs for å ha korte urler overalt.

Reply:
Skulle vi returnert noe annet enn tom string isåfall, bare for å gjøre det eksplisitt?

Reply:
Eg gjorde det berre likt som path-resolver rett over. Dessuten forventer Movie at path ikkje er undefined, så lar det være slik for ikkje å knekke ndla-film.